Composer/packages/lib/shared/src/skillsUtils/index.ts

```diff
@@ -13,8 +20,8 @@ export function fetchFromSettings(path: string, settings: DialogSetting): string
   return '';
 }
 
-export const convertSkillsToDictionary = (skills: Skill[]) => {
-  const mappedSkills = skills.map(({ msAppId, endpointUrl, manifestUrl, name }: Skill) => {
+export const convertSkillsToDictionary = (skills: any[]) => {
```

Comment:
We should try to avoid using `any` here

Reply:
So the reason i had to use `any` here is because the skills type has changed now. This conversion happens during migration of old bots which has a skill object with manifestUrl name etc. Those fields are not present anymore in the interface.